tests/generic_config_updater/gu_utils.py

```diff
@@ -11,6 +10,7 @@
 
 CONTAINER_SERVICES_LIST = ["swss", "syncd", "radv", "lldp", "dhcp_relay", "teamd", "bgp", "pmon", "telemetry", "acms"]
 DEFAULT_CHECKPOINT_NAME = "test"
+GCU_SONIC_DIR_PATH = "/usr/local/lib/python3.9/dist-packages/generic_config_updater"
```

Comment:
>python3.9 [](http://example.com/codeflow?start=37&length=9)

Hardcoded python3 version is going break in next Debian version. Suggest find the path programmatically.

```
$ python3
Python 3.9.2 (default, Feb 28 2021, 17:03:44)
[GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import generic_config_updater
>>> print(generic_config_updater.__path__)
['/usr/local/lib/python3.9/dist-packages/generic_config_updater']
```